### DISCUSSION TITLE
Сибсети, не работает

### DISCUSSION THREAD

**Discussion by votarum** [Anonym-tsk/nfqws-keenetic#68] [Делимся опытом]
Небольшой региональный провайдер. В том же городе на других операторах все замечательно работает. На этом - Сибсети - нет. Перебирал параметры, пока не нашел ключа. Единственное что заметил - провайдер раздает адреса абонентам из частной сети 10.*.*.*, значит трафик NATится где-то еще раз. Не знаю, может ли это так повлиять. На РТ натил и 2 и 3 раза, все работает. Подкинте идей куда копать.
p.s. Перестало работать на 2.2.1 сегодня утром. После обновления на 2.3.2 также не работает.

**Comment by berzerk54** [comment]
тоже сибсети, тоже перестал работать yt, другие нужные мне домены в целом работают нормально

**Comment by Adm1nistrator** [comment]
Новосибирск, Сиб Сети, Аналогично, все работало и вчера перестало, update на 2.3.2 результатов пока не дал, все остальные ресурсы работают отлично, youtube превью и комменты есть, видео не грузит. 

<img width="1853" alt="image" src="https://github.com/user-attachments/assets/6df3da0a-18f9-4b27-bec0-2478121deeeb">

**Comment by votarum** [comment]
Примечательно, что проблема совсем локальная - в Бердске на Сибсетях все работает с настройками по умолчанию.

**Comment by votarum** [comment]
есть уcпех для [goodbyedpi](https://ntc.party/t/%D0%B5%D1%81%D0%BB%D0%B8-%D0%BE%D0%BF%D1%8F%D1%82%D1%8C-%D0%BF%D0%B5%D1%80%D0%B5%D1%81%D1%82%D0%B0%D0%BB-%D0%B3%D1%80%D1%83%D0%B7%D0%B8%D1%82%D1%8C%D1%81%D1%8F-youtube-%D0%B8%D0%BB%D0%B8-%D0%B5%D0%B3%D0%BE-%D0%B2%D0%B8%D0%B4%D0%B5%D0%BE/10529/336), может быть поможет найти нужные опции для nfqws

**Comment by votarum** [comment]
Ютуб снова работает, самопроизвольно, ничего не делал

**Comment by Adm1nistrator** [comment]
Подтверждаю, у мен тоже заработал.


Сб, 28 сент. 2024 г. в 13:45, votarum ***@***.***>:
